lib/llm-events/aws-bedrock/bedrock-command.js

```diff
@@ -86,9 +86,9 @@ class BedrockCommand {
     } else if (this.isClaude3() === true) {
       result = this.#body?.messages?.reduce((acc, curr) => {
         if (typeof curr?.content === 'string') {
-          acc += curr?.content
-        } else if (Object.keys(curr?.content).length) {
-          acc += curr?.content.text ?? ''
+          return curr?.content
+        } else if (Array.isArray(curr?.content) && Object.keys(curr?.content[0]).length) {
+          return curr?.content[curr?.content.length - 1].text ?? ''
```

Comment:
we only care about the last message, which would be the most recent user prompt.

Reply:
If that is the case, do we need to retain the `.reduce`?

Reply:
Im thinking in the future we might want to use the reduce to filter out messages from `{type: text}` and `{type: image}` at this current point in time. I don't see a benefit with using the `reduce` function.

Reply:
How are other agent teams handling these cases? I haven't seen a spec that specifically calls out this model's operations.

Reply:
I think for 1, it's just a relatively new API that got introduced after we added Bedrock support. Python also seems to only handle the string case and not the multi-chunk case. I'm not positive but I'd guess that's the same in other agents (who support Bedrock) as well.

For 2, I think Claude is actually getting closer to how we handle Chat Completions in OpenAI [here](https://github.com/newrelic/node-newrelic/blob/57e6be9f4717fde3caada0e3ca3680959180f928/lib/instrumentation/openai.js#L139). For one completion call, we already split out several message events based on the context messages that were passed in